Runtime/MeshPool.cs

```diff
@@ -0,0 +1,46 @@
+using System;
+using UnityEngine;
+using UnityEngine.Pool;
+
+namespace CesiumForUnity
+{
+    public static class CesiumObjectPool
```

Comment:
Please move this to its own file. Also, let's mark it `internal`, because I don't think there's any particular reason for it to be used outside the CesiumRuntime assembly, and making it internal preserves our ability to change it in the future without breaking backward compatibility.